### ISSUE TITLE
IE11 fails to run Aurelia due to => not getting transpiled from TS to ES5

### ISSUE THREAD

**Issue by dmongrel** [aurelia/framework#914]
**I'm submitting a bug report**

* **Library Version:**
2.1.6

**Please tell us about your environment:**
* **Operating System:**
Windows 10

* **Node Version:**
11.1.0

* **NPM Version:**
6.1.0

* **Aurelia CLI OR JSPM OR Webpack AND Version**
CLI 1.0.0-beta.3

* **Browser:**
IE 11

* **Language:**
TypeScript 3.1.2 

**Current behavior:**
Aurelia fails to run in IE11.  The aurelia-binding function camelCase() has a => which does not get transpiled to es5/es2015.  In IE11, this causes a SCRIPT1002: Syntax error in vendor-bundle.js(6008,97) 

**Expected/desired behavior:**

The fat arrow (=>) should be transpiled and not appear in vendor bundle, because IE11 cannot handle them.
To reproduce, download aurelia-CLI, create a project, using  TypeScript, no CSS, max-minification.  My project has custom elements, so makes sure yours does as well.

* **What is the motivation / use case for changing the behavior?**
Aurelia does not work on IE11 at all anymore (at least the version I'm using)

**Comment by EisenbergEffect** [comment]
@dmongrel Can you update the CLI to beta.4. We just pushed a fix yesterday that I think is related to this. Let me know!

**Comment by dmongrel** [comment]
@EisenbergEffect Yes it seems to get through the errors related to transpiling to ES5.  I'm not sure if this would be classified as the same bug, but, after the b4 version, the aurelia-loader-default throws an error in ensureOriginOnExports.  It thinks that target is null or undefined when it reaches the if statement.   I can file a new bug if you prefer.

function ensureOriginOnExports(executed, name) {
  var target = executed;
  var key = void 0;
  var exportedValue = void 0;

  if (target.__useDefault) {
    target = target['default'];
  }

**Comment by EisenbergEffect** [comment]
@huochunpeng Are you available to look into this?

**Comment by huochunpeng** [comment]
@dmongrel can you try reset the cache? `au clear-cache`

@bigopon I guess you did not see this error when testing on ie11?

**Comment by bigopon** [comment]
Let me test this properly first. I'll update as soon as I finish that

**Comment by bigopon** [comment]
It works totally fine in my local env. Here is how it looks like on IE11

![image](https://user-images.githubusercontent.com/9994529/48186168-56bd3580-e38b-11e8-8819-c6e435c600d9.png)

With TS - Requirejs, for the following structure:

![image](https://user-images.githubusercontent.com/9994529/48186196-6c325f80-e38b-11e8-9a08-94a33cdfc5bb.png)

For the following app files:
![image](https://user-images.githubusercontent.com/9994529/48186224-81a78980-e38b-11e8-8269-ace2648c69f3.png)

**Comment by dmongrel** [comment]
OK, I think we can close this issue.  I tried a default project, and at least that works now.  Thanks.  Au clear-cache had no apparent effect on the issue, which appears to be something else.  I'll have to track it down (likely is plugin-related:  I'm using aurelia-jstree and aurelia-dialog).  Those are the only things that should really be different from the default project.

**Comment by dmongrel** [comment]
Turns out the default error comes up, when tsconfig.json has compilerOptions target es2015.  I changed it to es5 and that solved the remaining issue.  Please close this issue, everything is now running as it should be.  If you'd like me to file a bug regarding the compiler option, let me know.
